Userland/Libraries/LibWeb/Fetch/Infrastructure/FetchController.cpp

```diff
@@ -85,12 +85,39 @@ void FetchController::abort(JS::Realm& realm, Optional<JS::Value> error)
     if (!error.has_value())
         error = fallback_error;
 
-    // FIXME: 4. Let serializedError be StructuredSerialize(error). If that threw an exception, catch it, and let serializedError be StructuredSerialize(fallbackError).
-    // FIXME: 5. Set controller’s serialized abort reason to serializedError.
-    (void)error;
+    // 4. Let serializedError be StructuredSerialize(error). If that threw an exception, catch it, and let serializedError be StructuredSerialize(fallbackError).
+    auto serialized_error_result = HTML::structured_serialize(realm.vm(), error.value());
+    if (serialized_error_result.is_exception()) {
+        m_serialized_abort_reason = HTML::structured_serialize(realm.vm(), fallback_error).value();
```

Comment:
this should use the MUST() macro here, instead of grabbing the .value from the WebIDL::ExceptionOr.